### DISCUSSION TITLE
How works buckets for histograms

### DISCUSSION THREAD

**Discussion by let-skip-cake** [open-telemetry/opentelemetry-dotnet#5925] [Q&A]
Hello everyone
I have a question about how buckets work in OpenTelemetry. I haven’t been able to get the separation to function as expected. However, I’ve already raised this issue in a different thread here: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/2246

Would it be possible to continue the discussion here?

**Comment by TimothyMothra** [comment]
I'm going to close this discussion in favor of continuing the discussion on your linked issue.
